packages/server/src/presentation/controllers/logEntriesController.ts

```diff
@@ -12,7 +12,7 @@ logEntriesController.get('/logs/:logId/log-entries', async (req, res) => {
   res.json(logEntries);
 });
 
-logEntriesController.put('/logs/:logId/log-entries', async (req, res) => {
+logEntriesController.post('/logs/:logId/log-entries', async (req, res) => {
```

Comment:
I ended up changing this to a post route as it creates an entry in the db. Change it back if you think other clients may be using this still as a put.